openlabcmd/README.md

```diff
@@ -29,3 +29,58 @@ optional arguments:
   --nocolor      Enable the no color mode.
   --recover      Enable the auto recover mode.
 ```
+
+### ha
+openlab ha cluster management command.
+
+#### node
+
+CRUD for node
+
+* openlab ha node list
+* openlab ha node get
+  ```
+  usage: openlab ha node get [-h] name
+  positional arguments:
+    name        The node name.
+  ```
+* openlab ha node create
+  ```
+  usage: openlab ha node create [-h] --type {nodepool,zuul,zookeeper-only}
```

Comment:
the nodes data maybe initialized when deploy finished automatically.

Reply:
Hi @wangxiyuan , this command means to create Zookeeper node info for HA scenario, it is not deployment command, right? If it is init command, `ha node init` looks better

```
openlab ha node init --type nodepool --role master --ip 192.168.1.199 {cloud-provider}-openlab-nodepool
```

Reply:
Like @liu-sheng said, this command should be called by `labkeeper` when deploy finished automatically.

I'm hesitate to remote this cli since I'm not sure if operator would use it or not. Maybe we can just leave it here at meanwhile and revisit it in the future. I'll leave a warning to this command.

`init` is fine.